src/google-fonts/index.js

```diff
@@ -177,13 +177,13 @@ function GoogleFonts() {
 
 			function getFontData( event ) {
 				const font = event.detail.font;
-				const nameTable = font.opentype.tables.name;
+				const { name } = font.opentype.tables;
```

Comment:
This change is just to make this variable assignment consistent across the two font files.